plugins/extractors/bigquery/extractor.go

```diff
@@ -60,7 +59,7 @@ func (e *Extractor) Extract(ctx context.Context, config map[string]interface{},
 			break
 		}
 		if err != nil {
-			e.logger.Error(err)
+			e.logger.Error("failed to fetch, skipping dataset", err)
```

Comment:
this will not work as expected as the method takes the message and key/value pairs. What we should be doing is
```
e.logger.Error("failed to fetch, skipping dataset", "error", err)
```
or
```
e.logger.Error(errors.Wrap(err, "failed to fetch, skipping dataset").Error())
```
or
```
e.logger.Error(err.Error())
```

Reply:
made the change